tests/PySys/environments/environment_c8y.py

```diff
@@ -1,197 +1,34 @@
-import json
 import base64
+import json
```

Comment:
I feel we should have one environment for each cloud, more will be confusing.

Reply:
I would rather say we should have an environment for a specific use case. By using environments that can inherit from another one we can reduce code duplication in environments and tests.